server/src/main/java/com/hedera/block/server/producer/ProducerBlockItemObserver.java

```diff
@@ -50,21 +56,19 @@ public class ProducerBlockItemObserver implements StreamObserver<PublishStreamRe
      *     arrive from the upstream producer
      * @param publishStreamResponseObserver the response stream observer to send responses back to
      *     the upstream producer for each block item processed
-     * @param itemAckBuilder the item acknowledgement builder to use when sending responses back to
-     *     the upstream producer for each block item processed
      * @param serviceStatus the service status used to determine if the downstream service is
      *     accepting block items. In the event of an unrecoverable exception, it will be used to
      *     stop the web server.
      */
     public ProducerBlockItemObserver(
             @NonNull final Publisher<BlockItem> publisher,
-            @NonNull final StreamObserver<PublishStreamResponse> publishStreamResponseObserver,
-            @NonNull final ItemAckBuilder itemAckBuilder,
+            @NonNull
+                    final StreamObserver<com.hedera.hapi.block.protoc.PublishStreamResponse>
+                            publishStreamResponseObserver,
```

Comment:
Opinion:
This should look like the following.  If spotless doesn't allow that, we should change spotless until it does.
```suggestion
            @NonNull final StreamObserver<com.hedera.hapi.block.protoc.PublishStreamResponse>
                publishStreamResponseObserver,
```

Reply:
I added these details to [the ticket](https://github.com/hashgraph/hedera-block-node/issues/128).